lib/Data/Chronicle/Reader.pm

```diff
@@ -126,13 +125,30 @@ sub get {
     my $category = shift;
     my $name     = shift;
 
-    my $key = $category . '::' . $name;
+    my @cached_data = $self->mget([[$category, $name]]);
+    return $cached_data[0] if @cached_data;
+
+    return undef;
+}
+
+=head3 C<< my $data = mget([["category1", "name1"], ["category2", "name2"], ...]) >>
```

Comment:
FYI: I'm aware this `=head3` line is just matching style with the rest of the module, so don't feel you should change it.

But... it's generally best to have headings be simple method names like

    =head3 mget

and then include a one-line synopsis example right below it to demonstrate the syntax. Many of the HTML generation programs use that heading to generate HTML anchors, so if you stick to just `mget` then links of the form `L<Data::Chronicle::Reader/mget>` should work.

Probably a change to make wholesale across the module at another time.

Reply:
Fixed